scripts/provisioning/setupcmd.py

```diff
@@ -594,24 +584,43 @@ def get_config_param_for_BG_delete_account(self):
 
     return bgdelete_acc_input_params_dict
 
-  def delete_replication_config(self):
-    conn = ldap.initialize("ldapi://")
-    conn.sasl_non_interactive_bind_s('EXTERNAL')
-
-    dn = "cn=config"
-    self.deleteattribute(conn, dn, "olcServerID")
-
-    dn = "olcDatabase={0}config,cn=config"
-    self.deleteattribute(conn, dn, "olcSyncrepl")
-    self.deleteattribute(conn, dn, "olcMirrorMode")
-
-    dn = "olcDatabase={2}mdb,cn=config"
-    self.deleteattribute(conn, dn, "olcSyncrepl")
-    self.deleteattribute(conn, dn, "olcMirrorMode")
-
-  def deleteattribute(self, conn, dn, attr_to_delete):
-    mod_attrs = [(ldap.MOD_DELETE, attr_to_delete, None)]
-    try:
-      conn.modify_s(dn, mod_attrs)
-    except:
-      self.logger.info('Attribute '+ attr_to_delete + ' is not configured for dn '+ dn)
+  def modify_attribute(self, dn, attribute, value):
+        # Open a connection
+        ldap_conn = ldap.initialize("ldapi:///")
+        # Bind/authenticate with a user with apropriate rights to add objects
+        ldap_conn.sasl_non_interactive_bind_s('EXTERNAL')
+        mod_attrs = [(ldap.MOD_REPLACE, attribute, bytes(str(value), 'utf-8'))]
+        try:
+            ldap_conn.modify_s(dn, mod_attrs)
+        except:
+            self.logger.error('Error while modifying attribute- '+ attribute )
+            raise Exception('Error while modifying attribute' + attribute)
+        ldap_conn.unbind_s()
+
+  def search_and_delete_attribute(self, dn, attr_to_delete):
+        conn = ldap.initialize("ldapi://")
+        conn.sasl_non_interactive_bind_s('EXTERNAL')
+        ldap_result_id = conn.search_s(dn, ldap.SCOPE_BASE, None, [attr_to_delete])
```

Comment:
Separate this in different function getCountofRecords()

Reply:
Fixed